.env.example

```diff
@@ -0,0 +1,14 @@
+TEST_PRN="PESX20XX00XXX"
+TEST_PASSWORD="**********"
+TEST_SRN="PESXUGXXYYXXX"
+TEST_PROGRAM="Bachelor of Technology"
+TEST_SEMESTER="Sem-8"
+TEST_SECTION="Section F"
+TEST_EMAIL="********"
+TEST_PHONE="********"
+TEST_BRANCH="Computer Science and Engineering"
+TEST_BRANCH_SHORT_CODE="CSE"
+TEST_CAMPUS="EC"
+TEST_CAMPUS_CODE=2
+TEST_INSTITUTE_NAME="PES University" # should be deprecated
```

Comment:
Remove any deprecated fields. #28 will not include these any more, so future PRs should not have them either.